docs/farming.md

```diff
@@ -19,17 +20,46 @@ You can find these executables in the [Releases](https://github.com/subspace/sub
 
 > This is a ***non-incentivized*** testnet. Meaning there are no rewards in place at this time, and has absolutely no financial benefit to being run at this time.
 
-### A. To Farm With The Network (Online)
+<details><summary> 🖼️ Windows Instructions (Click to Expand)</summary>
+<p>
```

Comment:
Less HTML in Markdown is better. Do we really need `<p>`?

Reply:
> Less HTML in Markdown is better. Do we really need `<p>`?

after testing not in this instance (because GitHub's Markdown support is 👍) But most other places will require the <p> for proper inline markdown rendering, headers and such wont show etc. 

Will remove 😄